data/mods/Magiclysm/items/enchanted_belts.json

```diff
@@ -57,7 +57,6 @@
         "pocket_type": "CONTAINER",
         "max_contains_volume": "3 L",
         "max_contains_weight": "9 kg",
-        "max_item_length": "70 cm",
```

Comment:
why did you delete this stuff?

Reply:
I thought I didn't merge this stuff let me undo